### DISCUSSION TITLE
Keyboard input stopped working

### DISCUSSION THREAD

**Discussion by breadbyte** [ppy/osu#23655] [Q&A]
### Type

Game behaviour

### Bug description

Steps to reproduce:
1. Open the osu! configuration folder.
2. Go into `input.json` and replace the `osu.Framework.Input.Handlers.Keyboard.KeyboardHandler`'s `Enabled` flag to `false`.
3. Open osu!.
4. Textboxes should accept character key input, but not special keys (Ctrl, Alt, Function keys, Backspace, Enter, arrow keys, etc.)
4a. Inputs will not be accepted when trying to set keyboard bindings or attempting to play.

I have detailed a way to _reproduce_ the issue based on how I _fixed_ it, but I could not figure out _how_ this flag was set to `false` in the first place.

I was just opening osu!, and suddenly step 4 was happening to me. Going through the game's configuration, I have noticed that step 2 was set to `false`, and setting that back to `true` fixed the issue.

### Version

2023.513.0

### Logs

[performance.log](https://github.com/ppy/osu/files/11557452/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11557453/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11557454/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11557455/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11557456/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11557457/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11557458/updater.log)
[database.log](https://github.com/ppy/osu/files/11557459/database.log)
[input.log](https://github.com/ppy/osu/files/11557460/input.log)
[network.log](https://github.com/ppy/osu/files/11557461/network.log)

**Comment by bdach** [comment]
Obviously if you disable the keyboard handler, the keyboard is going to stop working.

Giving the reproduction steps provided is pretty useless because there's nothing to fix on that basis. If you can reproduce *how* to get into this broken state without tampering with configuration, then we can consider this a real issue.
